src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +43,36 @@
      * Андрей - 600
      * София - 900</p>
      */
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+
+        LocalDate dateFromObj = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate dateToObj = LocalDate.parse(dateTo, FORMATTER);
+        StringBuilder s = new StringBuilder("Отчёт за период " + dateFrom + " - " + dateTo + "\n");
```

Comment:
s - isn't informative name of the variable

Reply:
Ok, fixed.